### DISCUSSION TITLE
AI reads always 6 instead of 5

### DISCUSSION THREAD

**Discussion by koethino** [jomjol/AI-on-the-edge-device#3380] [Q&A]
Hello 
the AI reads always 6 instead of 5 at the digit ROIs! what can I do?
thanks

**Comment by SybexX** [comment]
@koethino Without sufficient information(Alignment/Placement of the ROIs, image quality, Configuration setting, model used.................), no one can help you.

https://github.com/jomjol/AI-on-the-edge-device/discussions/3371#discussioncomment-11147155
https://github.com/jomjol/AI-on-the-edge-device/discussions/3368
https://github.com/jomjol/AI-on-the-edge-device/discussions/3356#discussioncomment-11131828
